Source/ACE.Server/Program.cs

```diff
@@ -110,6 +110,9 @@ public static void Main(string[] args)
             log.Info("Initializing WorldManager...");
             WorldManager.Initialize();
 
+            log.Info("Preloading Global Event Landblocks...");
+            LandblockManager.PreloadGlobalEventLandblocks();
```

Comment:
There should be a config option for this

If I am testing non-event related stuff, I would prefer not to load Hebian-To

Reply:
Also, this should probably just be nixed completely, as there already exists a landblock preloading / permaload system

The existing landblock preloading system should be upgraded so the landblocks is pre-loads is determined by a config file, instead of those landblocks being hardcoded

Reply:
It's a pretty important system, but i can just make the default option be load instead of not load...

I disagree about removing completely as there should be two seperate things.. One is Hebian-To and possibly any other landblocks found to have global things on them, and then the custom or server op configs landblock loads for things like common towns or high-traffic blocks.. 

I'm not going to touch upon that for this fix.. just want to get the global event block loaded by default

Reply:
That's the purpose of the config files, and specifying which landblocks to permaload in there

The default server config file would have Hebian-To and any other landblocks that are needed. If the server op also wanted to permload other landblocks, such as all the towns, then they would add that to the config. ie. there would be a server.config with Hebian-To, a towns.config with all the towns, and the server op would specify which config files to load